var/spack/repos/builtin/packages/wps/package.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+import glob
+import tempfile
+from os.path import dirname,join
+import multiprocessing as mp
+
+class Wps(AutotoolsPackage):
```

Comment:
Should use `Package` base class and `phases = ['configure', 'build', 'install']`, as mentioned in WRF PR.

Reply:
ditto